### PR TITLE
add korean language

### PR DESCRIPTION
**Pull request by karriz-dev**
translate complete !

### CODE REVIEW

**Comment by AnthonyLaw** [comment]
I saw some of the messages are not translated.

Example this one and those `Error` message
image

**Comment by karriz-dev** [comment]
> I saw some of the messages are not translated.
> 
> Example this one and those `Error` message image

In the case of `HOME` in general Korean web services, it is expressed in English rather than Korean.   
Therefore, `HOME` has not been changed.

**Comment by karriz-dev** [comment]
and `ERROR` msg I missed 😢 

Thank you for announce. I update that part and re-commit.

File: src/config/i18n/ko.json [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+    "created": "생성됨",
+    "ago": "이전",
+    "messageTypeDescriptor_-1": "로우 메시지",
+    "messageTypeDescriptor_0": "일반 메시지",
+    "messageTypeDescriptor_1": "암호화된 메시지",
+    "messageTypeDescriptor_254": "위임 하베스팅 확인 메시지",
+    "votingNode": "투표 노드",
+    "nonVotingNode": "비투표 노드",
+    "votingNodeTooltips": "Sync ahead of common node.",
+    "nonVotingNodeTooltips": "Sync behind of Voting node.",
```
Comment (AnthonyLaw) [inline src/config/i18n/ko.json:375]: Do you like to keep these 2 in English as well?

Comment (karriz-dev) [reply src/config/i18n/ko.json:375]: Thank you so much for the detailed review.

**Review by AnthonyLaw — APPROVED** [review]
Great job!

**Pull request merged**